app/helpers/react_helper.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module ReactHelper
+  def search_results_layout(search, params, vertical, affiliate)
+    data = {
+      additionalResults: search.govbox_set,
+      locale: YAML.load_file("config/locales/#{affiliate.locale}.yml"),
+      noResultsMessage: (search.affiliate.no_results_error if search.results.blank? && search.query.present?),
+      params: params,
+      relatedSites: related_sites(affiliate.connections, search.query),
+      resultsData: search.normalized_results,
+      vertical: vertical
+    }
```

Comment:
```suggestion
    }.compact_blank
```

Reply:
We need to remove `nil`s and empty arrays in order to simplify the React code. Without `compact_blank` we'd have to use code like `props?.additionalResults?.jobs?.length > 0 && <Component />`. With `compact_blank` this simplifies to `props?.additionalResults?.jobs && <Component />`.

Reply:
Fixed here https://github.com/GSA/search-gov/pull/1288/files#diff-2095d5bfe192c31a5f3af38d7592bafc30cc9443a4c2f25034d6449ed5dcc0a6R15